<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <property name="spacing">
      <number>1</number>
     </property>
     <item row="5" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelFamily">
       <property name="minimumSize">
        <size>
         <width>410</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>410</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="comboBoxFamily">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Family00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Family01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Family02</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Family03</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Family04</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Params</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <property name="spacing">
         <number>1</number>
        </property>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Re</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Img</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>C:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="leCReal">
          <property name="text">
           <string>-0.74797</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="leCImg">
          <property name="text">
           <string>-0.072500001</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Q:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="leQReal">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="leQImg">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>N:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinBoxN">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>16</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Render</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <property name="spacing">
         <number>1</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBoxMandelbrot">
            <property name="text">
             <string>Mandelbrot</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxOrbitTrap">
            <property name="text">
             <string>Orbit trap</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonResetArea">
            <property name="text">
             <string>Default Area</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Max. Iters: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxMaxIters">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Radius:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leRadius">
            <property name="text">
             <string>2.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Cmap:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxCmaps">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>10</number>
            </property>
            <item>
             <property name="text">
              <string>gpGrayscale</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpHot</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpCold</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpNight</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpCandy</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpGeography</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpIon</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpThermal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpPolar</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpSpectrum</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpJet</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gpHues</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxLogScale">
            <property name="text">
             <string>LogScale</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <item row="1" column="4">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Distance</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3" colspan="2">
           <widget class="QComboBox" name="comboBoxOrbitType">
            <property name="currentIndex">
             <number>8</number>
            </property>
            <item>
             <property name="text">
              <string>x^2+y^2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>|x|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>|y|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>|x*y|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x^2+y^2-2*x*y</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x^2+y^2+2*x*y</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x^2+y^2+|2*x*y|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>x^2+y^2-|2*x*y|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Min(|x|,|y|)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Max(|x|,|y|)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Line Dist.</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>|x-y|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>|x+y|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>|x|+|y|</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>|x|-|y|</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leOrbitReal">
            <property name="text">
             <string>1.02</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="leOrbitImg">
            <property name="text">
             <string>0.78</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Re</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Orbit point: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Img</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="5">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pBSaveRawData">
       <property name="text">
        <string>Save Raw Data</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
